src/components/KRadio/KRadio.vue

```diff
@@ -73,7 +91,7 @@ const props = defineProps({
    */
   modelValue: {
     type: [String, Number, Boolean, Object],
-    default: 'on',
+    default: 'true',
```

Comment:
question: why are we changing the default here?

Reply:
I thought boolean `true` is more "conventional" than string with value of `'on'`.

Reply:
Ah yeah not sure why it's out of date here but I changed it to boolean `true`